source/playbooks/customize-a-playbook.rst

```diff
@@ -0,0 +1,77 @@
+Customize your playbook
```

Comment:
Can you add the appropriate subscription and deploy badges to this page, please?